org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/git/GitProvider.xtend

```diff
@@ -28,7 +40,15 @@ class GitProvider {
 		val workspace = workspaceProvider.workspace
 		return workspaceToGitCache.get(workspace)
 	}
-
+	
+	/**
+	 * configure transport commands with ssh credentials (if configured for this dropwizard app)
+	 */
+	def <T, C extends GitCommand<T>> GitCommand<T> configureTransport(TransportCommand<C, T> command) {
```

Comment:
this (currently?) only wraps a private method without adding anything by itself. If `setSshSessionFactory` is simplified (by factoring out the anonymous inner class, see below), maybe the two methods can be "merged". (I would keep the name `configureTransport` -> suggestion: remove this method, rename the other and make it public.)